Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -424,9 +424,12 @@ - (void)startWithOptions:(SentryReplayOptions *)replayOptions
                          options:replayOptions];
 }
 
-- (NSURL *)replayDirectory
+- (nullable NSURL *)replayDirectory
 {
     NSString *sentryPath = [SentryDependencyContainer.sharedInstance.fileManager sentryPath];
+    if (!sentryPath) {
+        return NULL;
```

Comment:
I doubt it matters functionally but I typically only use `NULL` for C/C++/ObjC++ and `nil` for plain ol objc

```suggestion
        return nil;
```